app/views/posts/index.html.erb

```diff
@@ -42,16 +45,11 @@
                       <%= image_tag(post.image, style: 'width:80%') %>
                     </p>
                   <% end %>
-                  <small>
-                    <a href="/posts/<%= post.id %>/like">Like</a>
-                    <br>
-                    <%= pluralize(post.likes,'like')%>
-                  </small>
+                  <%= button_to 'SUPRISE', post_likes_path(post), method: :post %>
+                  <p><%= post.likes.count %> <%= (post.likes.count) == 1 ? 'Like' : 'Likes'%></p>
```

Comment:
`<%= post.likes.count %> <%= (post.likes.count) == 1 ? 'Like' : 'Likes'%>`
could be refactored into
`pluralize(post.likes.count, 'like')`